connectors/.eslintrc.json

```diff
@@ -38,5 +39,8 @@
   "env": {
     "node": true,
     "es6": true
+  },
```

Comment:
Please handle in a separate PR along with implementation of deleteConnector as discussed on Slack